Units/parser-php.r/wp-guest.d/expected.tags

```diff
@@ -0,0 +1,14 @@
+top	input.php	/^<div id="top">$/;"	I	line:3	language:HTML	roles:def
+inner	input.php	/^ <div class="inner">$/;"	c	line:4	language:HTML	roles:attribute
+logo	input.php	/^<div id="logo">$/;"	I	line:6	language:HTML	roles:def
+pages	input.php	/^<div id="pages">$/;"	I	line:12	language:HTML	roles:def
+menubar	input.php	/^<ul class="menubar" id="menubar">$/;"	c	line:13	language:HTML	roles:attribute
+menubar	input.php	/^<ul class="menubar" id="menubar">$/;"	I	line:13	language:HTML	roles:def
+body	input.php	/^<div id="body">$/;"	I	line:26	language:HTML	roles:def
+content	input.php	/^  <main id="content">$/;"	I	line:27	language:HTML	roles:def
+post	input.php	/^  <article class="post" id="post-<?p/;"	c	line:31	language:HTML	roles:attribute
+post-<?p	input.php	/^  <article class="post" id="post-<?p/;"	I	line:31	language:HTML	roles:def
```

Comment:
I have a follow-up to fix the HTML promise range from the PHP lexer to fix this spurious `<?p`, but it currently depends on these changes so I'll post it after this is sorted out (no pressure, just saying that indeed, this tag is not exactly as it should be).

Reply:
Could you paste this comment into the commit log for this commit?
So people can understand this commit from its log only with 'git log'.